components/ThemeRegistry/theme.ts

```diff
@@ -52,8 +56,19 @@ const theme = createTheme({
     allVariants: {
       color: "#171B1E",
     },
+    button: {
+      fontSize: 12,
+      fontWeight: 700,
+      textTransform: "none",
```

Comment:
@alexfauquette any idea how to remove uppercase on secondary buttons but keep it on the others (primary, tabs...)?

Reply:
You can apply this style to the className `containedSecondary` class: https://mui.com/material-ui/api/button/#Button-css-containedSecondary

It exists also for the other variants

Reply:
So should I also move the `fontSize` and `fontWeight` in the style override?
I am not sure how to style the `containedSecondary` className properly (other than in a .css file).

Reply:
You have multiple choices

https://mui.com/material-ui/customization/theme-components/#theme-style-overrides

Usually you can write a CSS selector as a key, using `&` to target the current element.

As an example, you have in the codebase the following code. The same type of property can be use in styleOverride

```
sx={{
   [`& .${timelineOppositeContentClasses.root}`]: {
        flex: 0.2,
      },
    }}
```

But I found an easier one for your issue. I just commit

Reply:
Nice! 
I added the `fontSize` and `fontWeight` to the fonction in the following commit 👇 .

It felt weird to not to use the global theme for buttons, as those will (probably) be reused everywhere.